src/Configuration/Areas.php

```diff
@@ -23,7 +23,10 @@ public function __construct(Config $config)
         $this->initAreas();
     }
 
-    public function get($area, $key = null)
+    /**
+     * @return Collection|string
+     */
+    public function get(string $area, ?string $key = null)
```

Comment:
This change is fine for now, but we should strive to have only one return type, so we can use `: Collection` instead.